lib/pengine/clone.c

```diff
@@ -279,6 +279,24 @@ short_print(char *list, const char *prefix, const char *type, const char *suffix
     }
 }
 
+static void
+pe__short_output_text(pcmk__output_t *out, char *list, const char *prefix, const char *type, const char *suffix, long options)
+{
+    if(suffix == NULL) {
+        suffix = "";
+    }
+
+    if (list) {
+        fprintf(out->dest, "%s%s: [%s ]%s", prefix, type, list, suffix);
+
+        if (options & pe_print_suppres_nl) {
+            /* nothing */
+        } else if ((options & pe_print_printf) || (options & pe_print_ncurses)) {
+            fprintf(out->dest, "\n");
```

Comment:
We can drop the checks for pe_print_printf and pe_print_ncurses -- this will only be called for pe_print_printf

Basically pe_print_{log,html,curses,printf,xml} are being replaced with the new output model, while the other pe_print_* options are still valid

Reply:
Fixed.

Reply:
I'm having trouble finding what changed ...

Reply:
Oops. I checked all the changed functions if they have `pe_print_printf` and `pe_print_ncurses` checks. Now, they don't have them.